test/test_context.py

```diff
@@ -192,6 +196,37 @@ def test_cartocontext_table_exists(self):
         with self.assertRaises(NameError):
             cc._table_exists(self.test_read_table)
 
+    def test_cartocontext_delete(self):
+        """CartoContext.delete"""
+        cc = cartoframes.CartoContext(base_url=self.baseurl,
+                                      api_key=self.apikey)
+        data = {'col1': [1,2,3],
+                'col2': ['a','b','c']}
+        schema = {'col1': int,
+                  'col2': 'object'}
+        df = pd.DataFrame(data).astype(schema)
```

Comment:
Is the `.astype(schema)` necessary?

Reply:
No, but I was copying your unittest format for writing tables above

Reply:
In this case the schema isn't important. I believe there were tests for the `cc.write to test that the schema was set correctly.